lib/interactive.dart

```diff
@@ -289,6 +316,17 @@ class Interactive {
     return legendWrapper;
   }
 
+  html.DivElement _renderMessages() {
+    var messagesWrapper = html.DivElement()..classes = [MESSAGES_WRAPPER_CSS];
```

Comment:
consider `MESSAGES_WRAPPER_CSS_CLASS` - if I understand correctly it's not the full CSS that will get added here but the names of the class?

Reply:
Fixed all occurrences of css to css class